.github/workflows/rpe_test.yml

```diff
@@ -111,7 +110,7 @@ jobs:
       fail-fast: false
       matrix:
         os: 
```

Comment:
Why you removed this comment?

Reply:
@NadeemYaseen I will add the comments again since I shall re-push the entire file by doing a new PR, I need your comments/approval on the steps to install python on virtual environment, especially from line 238 to line 255.